src/main_benchmark.cpp

```diff
@@ -218,8 +221,100 @@ void triggerInferenceInALoop(
     averageWholeLatency = std::accumulate(latenciesWhole.begin(), latenciesWhole.end(), 0) / (double(niterPerThread) * 1'000);
     averagePureLatency = std::accumulate(latenciesPure.begin(), latenciesPure.end(), 0) / (double(niterPerThread) * 1'000);
 }
+
+void triggerInferenceInALoopResetBuffer(
+    std::future<void>& startSignal,
+    std::promise<void>& readySignal,
+    const size_t niterPerThread,
+    size_t& wholeThreadTimeUs,
+    double& averageWholeLatency,
+    double& averagePureLatency,
+    OVMS_Server* server,
+    OVMS_InferenceRequest* request,
+    const std::string& inputName,
+    std::uint32_t elementsCount) {
+    OVMS_InferenceResponse* response{nullptr};
+    std::vector<uint64_t> latenciesWhole(niterPerThread);
+    std::vector<uint64_t> latenciesPure(niterPerThread);
+    std::vector<std::vector<float>> preparedData;
+    for (int i = 0; i < niterPerThread; i ++) {
+        float random = ((float) rand()) / (float) RAND_MAX;
```

Comment:
I don't see `seed` being used, meaning we will get the same random values in every app run

Reply:
There is C++17 utility for generating random numbers